src/par/include/par/PartitionMgr.h

```diff
@@ -75,8 +75,7 @@ enum GraphType : uint8_t
 {
   CLIQUE,
   HYBRID,
-  STAR,
-  HYPERGRAPH
```

Comment:
Why remove HYPERGRAPH?

Reply:
possibly, but there is a lot of effectively dead code in this module